Page/PageServiceManager.php

```diff
@@ -126,7 +126,8 @@ protected function createResponse(PageInterface $page)
     {
         if ($page->getTarget()) {
             $page->addHeader('Location', $this->router->generate($page->getTarget()));
-            $response = new Response('', 302, $page->getHeaders() ?: array());
+            $redirectType = $page->getTargetRedirectType() ? $page->getTargetRedirectType() : 302;
```

Comment:
Maybe use this constant? https://github.com/symfony/symfony/blob/3bbb657560615b80c8a9b3b6da8602349bd3fbd9/src/Symfony/Component/HttpFoundation/Response.php#L36